schema.sql

```diff
@@ -7,4 +7,21 @@ CREATE TABLE animals(
   weight_kg DECIMAL (5, 2) NOT NULL,
 )
 
-ALTER TABLE animals ADD species VARCHAR(255);
\ No newline at end of file
+ALTER TABLE animals ADD species VARCHAR(255);
+
+CREATE TABLE owners (
+  id SERIAL PRIMARY KEY,
+  full_name TEXT,
+  age INTEGER
+);
+
+CREATE TABLE species (
+  id SERIAL PRIMARY KEY,
+  name TEXT
+);
+
+ALTER TABLE animals DROP COLUMN species;
+
+ALTER TABLE animals ADD COLUMN species_id INTEGER;
+
+ALTER TABLE animals ADD COLUMN owner_id INTEGER;
```

Comment:
Good work adding columns to the animal's table 💯 

- As it's on the requirements for this project, please add the corresponding foreign key referencing owners table `owners(id)`